drivers/include/at86rf2xx.h

```diff
@@ -92,6 +92,21 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Enable frame retry counter reporting
+ *
+ * The AT86RF2XX_HAVE_RETRIES flag enables support for NETOPT_TX_RETRIES NEEDED
+ * operation. Required for this functionality is the XAH_CTRL_2 register which
+ * contains the frame retry counter. Only the at86rf232 and the at86rf233
+ * support this register.
+ *
+ * @{
```

Comment:
you don't need a doc grouping here, move the docu directly above the `#define AT86RF2XX_HAVE_RETRIES`, i.e., within the `#if ... #endif`. Thus, please remove the doc grouping `@{`  and `/** @} */`, IMO not needed here, as it is only one define.